test/test_units.py

```diff
@@ -26,8 +26,27 @@ def test_quadratic_conversion():
         quadratic_conversion.physics_to_machine(2.5)
 
 
-def test_ppconversion():
+def test_ppconversion_to_physics_2_points():
     pchip_uc = UcPchip([1, 3], [1, 3])
     assert pchip_uc.machine_to_physics(1) == 1
     assert pchip_uc.machine_to_physics(2) == 2
     assert pchip_uc.machine_to_physics(3) == 3
+
+
+def test_pp_conversion_to_physics_3_points():
+    pchip_uc = UcPchip([1, 3, 5], [1, 3, 6])
+    assert pchip_uc.machine_to_physics(1) == 1
+    assert round(pchip_uc.machine_to_physics(2), 4) == 1.8875
```

Comment:
The problem here is that the returned value is a 1D numpy array.  You can't use `round()` on a numpy array.  But you can use `numpy.round()` instead.